backend/entityservice/tasks/project_cleanup.py

```diff
@@ -17,14 +18,15 @@ def remove_project(project_id):
     log = logger.bind(pid=project_id)
     log.debug("Remove all project resources")
 
-    conn = db.connect_db()
-    log.debug("Deleting project resourced from database")
-    db.delete_project_data(conn, project_id)
-    log.debug("Getting object store files associated with project from database")
-    object_store_files = db.get_all_objects_for_project(conn, project_id)
-    log.debug(f"Removing {len(object_store_files)} object store files associated with project.")
-    delete_minio_objects.delay(object_store_files, project_id)
-    log.info("Project resources removed")
+    with DBConn() as conn:
+        #conn = db.connect_db()
```

Comment:
we don't need that line any more. It doesn't spark joy. Thank it for all the services it has done for us and then send it of to new adventures.